plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1733,7 +1733,7 @@ public Object[][] fixedJoinDistributionTypes()
         return new Object[][] {{BROADCAST}, {PARTITIONED}};
     }
 
-    @Test(timeOut = 60_000, dataProvider = "fixedJoinDistributionTypes")
```

Comment:
The reason we have timeouts here is that these tests involve waiting on dynamic filters to be collected.
If there is a bug in DF colllection, the test would get stuck without an explicit timeout.
We can override the test with a bigger timeout in the derived class if some connector really needs it.
cc: @sopel39

Reply:
By default, we don't wait for DFs in data lake connectors. In JDBC connectors we wait with a short timeout (~10s). For some tests we specifically set the timeout to 1h to get predictability in collection of DF.
We've tried to consistently put timeouts for such cases, e.g. TestDeltaLakeDynamicFiltering, AbstractTestCoordinatorDynamicFiltering, BaseDynamicPartitionPruningTest. The only place we haven't done that is some implementations of AbstractTestJoinQueries.
If the test getting stuck in case of a bug in DF collection is not a big deal and we'll be able to figure out the problem from test reports anyway, then I'm ok with this change.

Reply:
> If the test getting stuck in case of a bug in DF collection is not a big deal and we'll be able to figure out the problem from test reports anyway, then I'm ok with this change.

Will we easily know which test deadlocks??